### PR TITLE
Update Heroku documentation

### PR DESCRIPTION
**Pull request by cneijenhuis**
Current Postgres driver for 9.4.1
Non-deprecated evolutions setting

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @cneijenhuis,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: documentation/manual/detailedTopics/production/cloud/ProductionHeroku.md [hunk 1]
```diff
@@ -177,13 +177,13 @@ $ heroku logs -a obscure-sierra-7788
...
 ```
 
 Then create a new file in your project's root directory named `Procfile` (with a capital "P") that contains the following (substituting the `myapp` with your project's name):
 
 ```txt
-web: target/universal/stage/bin/myapp -Dhttp.port=${PORT} -DapplyEvolutions.default=true -Ddb.default.driver=org.postgresql.Driver -Ddb.default.url=${DATABASE_URL}
+web: target/universal/stage/bin/myapp -Dhttp.port=${PORT} -Dplay.evolutions.db.default.autoApply=true -Ddb.default.driver=org.postgresql.Driver -Ddb.default.url=${DATABASE_URL}
 ```
 
 This instructs Heroku that for the process named `web` it will run Play and override the `applyEvolutions.default`, `db.default.driver`, and `db.default.url` configuration parameters.  Note that the `Procfile` command can be maximum 255 characters long.  Alternatively, use the `-Dconfig.resource=` or `-Dconfig.file=` mentioned in [[production configuration|ProductionConfiguration]] page.
```
Comment (marcospereira) [inline documentation/manual/detailedTopics/production/cloud/ProductionHeroku.md]: Please, update this instance of `applyEvolutions.default`.

Comment (cneijenhuis) [inline documentation/manual/detailedTopics/production/cloud/ProductionHeroku.md]: Sorry, I oversaw that!

**Comment by marcospereira** [comment]
Great! 

@cneijenhuis could you please squash the two commits? After that, this PR will be ready to be merged.

Thanks.

**Comment by cneijenhuis** [comment]
@marcospereira I just squashed the commits, so all should be good now :wink:

Thanks and have a nice weekend!

**Comment by marcospereira** [comment]
Backport here: 71cf13f

**Pull request merged**